.github/workflows/dataflow_engine_e2e.yaml

```diff
@@ -44,18 +44,18 @@ jobs:
         with:
           tail: '100'
 
-      - name: Collect docker logs on failure
-        if: ${{ failure() }}
-        uses: jwalton/gh-docker-logs@v2
-        with:
-          dest: 'logs'
+      # - name: Collect docker logs on failure
+      #   if: ${{ failure() }}
+      #   uses: jwalton/gh-docker-logs@v2
+      #   with:
+      #     dest: '/tmp/tiflow_engine_test/'
 
       - name: Upload logs to GitHub
-        if: ${{ failure() }}
+        if: ${{ always() }}
         uses: actions/upload-artifact@master
         with:
           name: basic-workflow-logs
-          path: logs/*
+          path: /tmp/tiflow_engine_test/*
```

Comment:
Should this change be applied to other cases, including node-failure-workflow, Worker-error-workflow, etc.

Reply:
Good suggestion, will test this later.